/retest is broken

Looks like either the wrong credentials are passed?!

```
Run python3 snapshot_manager/main.py \
  python3 snapshot_manager/main.py \
    --github-repo "${GITHUB_REPOSITORY}" \
    retest \
    --trigger-comment-id 3446060478 \
    --issue-number 1729 \
    --chroots fedora-rawhide-x86_64
  shell: /usr/bin/bash -e {0}
  env:
    chroots: fedora-rawhide-x86_64
    pythonLocation: /opt/hostedtoolcache/Python/3.12.11/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.12.11/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.11/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.11/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.11/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.11/x64/lib
    GITHUB_TOKEN: ***
[25/Oct/2025 07:32:04] INFO [github_util.py:59 __init__] Reading Github token from this environment variable: GITHUB_TOKEN
Traceback (most recent call last):
  File "/home/runner/work/llvm-snapshots/llvm-snapshots/snapshot_manager/main.py", line 446, in <module>
    main()
  File "/home/runner/work/llvm-snapshots/llvm-snapshots/snapshot_manager/main.py", line 83, in main
    SnapshotManager(config=cfg).retest(
  File "/home/runner/work/llvm-snapshots/llvm-snapshots/snapshot_manager/snapshot_manager/snapshot_manager.py", line 57, in retest
    repo = self.github.github.get_repo(self.config.github_repo)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/github/MainClass.py", line 479, in get_repo
    headers, data = self.__requester.requestJsonAndCheck("GET", url)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/github/Requester.py", line 623, in requestJsonAndCheck
    return self.__check(
           ^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/github/Requester.py", line 853, in __check
    raise self.createException(status, responseHeaders, data)
github.GithubException.BadCredentialsException: 401 {"message": "Bad credentials", "documentation_url": "https://docs.github.com/rest", "status": "401"}
Error: Process completed with exit code 1.
```

--- a/.github/workflows/retest-chroots-on-testing-farm.yml
+++ b/.github/workflows/retest-chroots-on-testing-farm.yml
@@ -79,7 +79,7 @@ jobs:
       - name: Run retest script
         shell: bash -e {0}
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TEST_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           chroots: ${{ env.chroots }}
         run: |
           python3 snapshot_manager/main.py \

